storage/backend/ceph/ceph.go

```diff
@@ -82,16 +82,20 @@ func (c *Driver) unlockAndLog(do storage.DriverOptions) {
 // Format formats a created volume.
 func (c *Driver) Format(do storage.DriverOptions) error {
 	device, err := c.mapImage(do)
-	if err != nil {
+	if err == ErrLockFailed {
```

Comment:
let me look at this patch again. I don't think it's very sound and I created it in a rush.